Documentation/guides/InstallBuildsFromMain.md

```diff
@@ -0,0 +1,78 @@
+# Installing builds from the `main` branch
+
+Each commit from xamarin-android/main (that builds successfully) is
+pushed to the [dotnet7 feed][0]. We can install these builds using
+`dotnet workload` commands.
+
+## Install the .NET SDK
+
+Begin by installing a nightly build of the .NET SDK from the
+[dotnet/installer repo][1]. See the `Microsoft.Dotnet.Sdk.Internal`
+version found in [`Version.Details.xml`][2], if you want to get an
+exact matching build. If you pick at least the same preview/release
+number, it should be *close enough* to work.
+
+You can guess what the URL is, if you need a specific version like
+`7.0.100-preview.6.22277.6`, for example:
+
+* Windows x64: https://dotnetbuilds.azureedge.net/public/Sdk/7.0.100-preview.6.22277.6/dotnet-sdk-7.0.100-preview.6.22277.6-win-x64.exe
+* Windows arm64: https://dotnetbuilds.azureedge.net/public/Sdk/7.0.100-preview.6.22277.6/dotnet-sdk-7.0.100-preview.6.22277.6-win-arm64.exe
+* macOS x64: https://dotnetbuilds.azureedge.net/public/Sdk/7.0.100-preview.6.22277.6/dotnet-sdk-7.0.100-preview.6.22277.6-osx-x64.pkg
+* macOS arm64: https://dotnetbuilds.azureedge.net/public/Sdk/7.0.100-preview.6.22277.6/dotnet-sdk-7.0.100-preview.6.22277.6-osx-arm64.pkg
+
+*See [WindowsOnArm64.md][3] for further details about Windows on Arm64.*
+
+## Install the `android` Workload
+
+On Windows, in a terminal running as Administrator run:
+
+```dotnetcli
+> dotnet workload install android --source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet7/nuget/v3/index.json --source https://api.nuget.org/v3/index.json
+...
+Successfully installed workload(s) android.
+```
+
+On macOS, you can prepend `sudo` to the same command.
+
+## Install Android API 32
+
+If you get the error:
+
+```
+error XA5207: Could not find android.jar for API level 32.
+This means the Android SDK platform for API level 32 is not installed.
+Either install it in the Android SDK Manager (Tools > Android > Android SDK Manager...),
+or change the Xamarin.Android project to target an API version that is installed.
+```
+
+Open the `Android SDK Manager` as described, click the gear icon in the
```

Comment:
Instead of/in addition to using the Android SDK Manager, we should suggest using the `InstallAndroidDependencies` target, which may require setting the `$(AndroidManifestType)` MSBuild property to `GoogleV2` (64b22fc94e453f77817b29e00afddff598336937):

```
# default SDK Manifest
dotnet build -t:InstallAndroidDependencies

# -or-

dotnet build -t:InstallAndroidDependencies -p:AndroidManifestType=GoogleV2
```

We may also want to mention setting `$(p:AcceptAndroidSDKLicenses)`=True.